core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -103,6 +106,24 @@ var TopicsAndSkillsDashboardPage = function() {
     });
   };
 
+  this.assignSkillWithIndexToTopicByTopicName = function(index, topicName) {
```

Comment:
Changing `index` to `skillIndex` would make the parameter names clearer and parallel in structure

Reply:
Updated